ui/components/multichain-accounts/multichain-account-list/multichain-account-list.tsx

```diff
@@ -120,6 +126,12 @@ export const MultichainAccountList = ({
 
         const groupsItems = Object.entries(walletData.groups || {}).flatMap(
           ([groupId, groupData]) => {
+            // If prop is provided, attempt render balance. Otherwise do not render balance.
+            const balanceText = formattedAccountGroupBalancesByWallet
+              ? (formattedAccountGroupBalancesByWallet?.[walletId]?.[groupId] ??
+                '$ n/a')
+              : '';
```

Comment:
needs confirmation - do we really want to show 'n/a' all the time as a fallback? Esp if this prop isn't passed in?

Reply:
We shouldn't use `$ n/a`. As @david0xd mentioned, this was just a placeholder during development, we should fallback to whatever value we use across the app for balances so we'll take your suggestion on this one @Prithpal-Sooriya.